apps/tangle-dapp/components/charts/IndependentRoleDistributionChart.tsx

```diff
@@ -55,22 +51,38 @@ const ProportionPieChart: FC<ProportionPieChartProps> = ({
           'flex flex-col items-center gap-1'
         )}
       >
-        {title && <Typography variant="body2">{title}</Typography>}
-        {showTotal && (
-          <Typography
-            variant="body1"
-            fw="bold"
-            className="text-mono-200 dark:text-mono-0"
-          >
-            {getRoundedAmountString(
-              data.reduce((acc, item) => acc + item.value, 0)
-            )}{' '}
-            {unit}
-          </Typography>
-        )}
+        <Typography variant="body2">{title}</Typography>
+        <Typography
+          variant="body1"
+          fw="bold"
+          className="text-mono-200 dark:text-mono-0"
+        >
+          {getRoundedAmountString(
+            data.reduce((acc, item) => acc + item.value, 0)
```

Comment:
If this is an amount derived from bigint (BN), eventually will need to be working with BN instead of numbers. Numbers can't hold all BN values.